### PR TITLE
Issue123 - Add Huobi DM exchange

### PR DESCRIPTION
**Pull request by glgnohk**

### CODE REVIEW

File: cryptofeed/exchange/huobi_dm.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+from cryptofeed.defines import HUOBI_DM, BUY, SELL, TRADES, BID, ASK, L2_BOOK
+from cryptofeed.exchange.huobi import Huobi
+from cryptofeed.standards import pair_std_to_exchange, pair_exchange_to_std, timestamp_normalize
+
+
+LOG = logging.getLogger('feedhandler')
+
+
+class HuobiDM(Huobi):
```
Comment (bmoscon) [inline cryptofeed/exchange/huobi_dm.py]: it doesnt look like this class shares anything with the parent class (Huobi). Shouldnt this just inherit from `Feed`?

**Comment by bmoscon** [comment]
also, i committed a fix to master that should resolve the CI issue, so the only thing i think needs to be fixed is the parent class for HuobiDM

File: cryptofeed/exchange/huobi_dm.py (hunk 1 above)
Comment (glgnohk) [reply cryptofeed/exchange/huobi_dm.py]: You are right. I originally used parent class, but then after testing, wound up changing every function. Looks like you already changed this. Thanks.

**Comment by glgnohk** [comment]
> also, i committed a fix to master that should resolve the CI issue, so the only thing i think needs to be fixed is the parent class for HuobiDM

What was the change for CI (environment variables missing?)? I didn't see this in the commits.

**Comment by bmoscon** [comment]
@glgnohk it was to the setup.py file. I specify the test dir now for the tests (which i didnt before). The file you committed with `test` in the name was being picked up as test code by pytest, which was of course, in error

**Comment by xiandong79** [comment]
@glgnohk @bmoscon  error in the cryptostore!!!

in redis: 
```
127.0.0.1:6379> KEYS '*'
 1) "l2_book-HUOBI_DM-ETH190927"
 2) "trades-HUOBI_DM-ETH190830"
 3) "trades-HUOBI_DM-BTC190830"
 4) "trades-HUOBI_DM-BTC190927"
 5) "trades-HUOBI_DM-ETH190906"
 6) "l2_book-HUOBI_DM-ETH190830"
 7) "l2_book-HUOBI_DM-ETH190906"
 8) "l2_book-HUOBI_DM-BTC190830"
 9) "trades-HUOBI_DM-BTC190906"
10) "l2_book-HUOBI_DM-BTC190906"
11) "trades-HUOBI_DM-ETH190927"
12) "l2_book-HUOBI_DM-BTC190927"
```

However, in cryptostore, cryptostore wants to save the 
```
019-08-26 15:14:38,914 : INFO : No data for HUOBI_DM-l2_book-ETH_NW
2019-08-26 15:14:38,914 : INFO : Reading HUOBI_DM-l2_book-ETH_CQ
2019-08-26 15:14:38,915 : INFO : No data for HUOBI_DM-l2_book-ETH_CQ
2019-08-26 15:14:38,915 : INFO : Reading HUOBI_DM-trades-BTC_CW
2019-08-26 15:14:38,915 : INFO : No data for HUOBI_DM-trades-BTC_CW
2019-08-26 15:14:38,915 : INFO : Reading HUOBI_DM-trades-BTC_NW
2019-08-26 15:14:38,915 : INFO : No data for HUOBI_DM-trades-BTC_NW
2019-08-26 15:14:38,915 : INFO : Reading HUOBI_DM-trades-BTC_CQ
2019-08-26 15:14:38,915 : INFO : No data for HUOBI_DM-trades-BTC_CQ
2019-08-26 15:14:38,916 : INFO : Reading HUOBI_DM-trades-ETH_CW
2019-08-26 15:14:38,916 : INFO : No data for HUOBI_DM-trades-ETH_CW
2019-08-26 15:14:38,916 : INFO : Reading HUOBI_DM-trades-ETH_NW
2019-08-26 15:14:38,916 : INFO : No data for HUOBI_DM-trades-ETH_NW
```

**Pull request merged**